docs/source/changelog.rst

```diff
@@ -10,6 +10,7 @@ Changelog
     * Changes
         * Make DFS match ``TimeSince`` primitive with all ``Datetime`` types (:pr:`1048`)
         * Change default branch to ``main`` (:pr:`1038`)
+        * Check input variable type for ``.delete_variables()`` (:pr:`1064`)
```

Comment:
Let's update this a bit to say:

Raise `TypeError` if improper input is supplied to ``Entity.delete_variables()`